src/plugins/saved_objects_management/public/management_section/objects_table/components/relationships.tsx

```diff
@@ -298,7 +298,7 @@ export class Relationships extends Component<RelationshipsProps, RelationshipsSt
             icon: 'inspect',
             'data-test-subj': 'relationshipsTableAction-inspect',
             onClick: (object: SavedObjectWithMetadata) => goInspectObject(object),
-            available: (object: SavedObjectWithMetadata) => !!object.meta.editUrl,
+            available: (object: SavedObjectWithMetadata) => !!(object.type && object.id),
```

Comment:
The inspect action is now available for all saved object types (we check that the object provided has a type and id to make sure that routing to the Inspect page will be valid). It used to only be available for saved objects that have an `editUrl` defined but is not the case anymore.